### ISSUE TITLE
Google Analytics on Handbook site

### ISSUE THREAD

**Issue by mtorres253** [18F/hub#560]
So that we can see all the wonderful 18Fers who are visiting the handbook, and what they are doing, we should be able to do this in Google Analytics!

**Comment by mbland** [comment]
This might be a five-minute task... Want to just knock it out? It should be a matter of adding this to the footer template:

```
{% guides_style_18f_include analytics.html %}
```

And this in the `_config.yml`:

```yaml
google_analytics_ua: UA-48605964-19
```

**Comment by mtorres253** [comment]
I'll take a look at it @mbland

**Comment by mtorres253** [comment]
Pull request filed in Handbook repo https://github.com/18F/handbook/pull/592

**Comment by andrewmaier** [comment]
Fixed per #592
